TicketSwap/solar
# Export TS types & enums

# Description

This PR will fix https://github.com/TicketSwap/solar/issues/1042. TLDR: We didn't export our enums and types that you need to use to provide certain props to our components. This will fix that.

## Changes


## How to test

- Checkout
- Check `yarn dev`
- Checkout `yarn storybook`
- Best way to test this is to use this in a TS project and try to use the Button component with the ButtonVariant enum
- Verify that it's working as expected
